tools/run_tests/distribtest_targets.py

```diff
@@ -232,6 +238,6 @@ def targets():
           ] + [
             NodeDistribTest('linux', 'x64', os, version)
```

Comment:
macos should go into "platform" arg, not to "docker_suffix".  As docker_suffix, you probably need to accept "None" in the constructor. (Just as I did for csharp).